### ISSUE TITLE
[Crash Report] Notification cannot be sheduled on Android 12+

### ISSUE THREAD

**Issue by Racooder** [SapuSeven/BetterUntis#369]
[logs-1678377682317.zip](https://github.com/SapuSeven/BetterUntis/files/10933606/logs-1678377682317.zip)

**Additional information**

- Android version: _12_
- BetterUntis version: _3.5.1 (34)_
- Installation source: _com.android.vending_
- Device: OnePlus Nord

**Comment by lm41** [comment]
```
java.lang.IllegalArgumentException: com.sapuseven.untis: Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent.
Strongly consider using FLAG_IMMUTABLE, only use FLAG_MUTABLE if some functionality depends on the PendingIntent being mutable, e.g. if it needs to be used with inline replies or bubbles.
	at android.app.PendingIntent.checkFlags(PendingIntent.java:378)
	at android.app.PendingIntent.getBroadcastAsUser(PendingIntent.java:648)
	at android.app.PendingIntent.getBroadcast(PendingIntent.java:635)
	at com.sapuseven.untis.receivers.AutoMuteSetup.b(:21)
	at d4.b.w(Unknown Source:14)
	at y3.b$c.e(:22)
	at q4.a.j(:4)
	at j7.f0.run(:2)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:233)
	at android.os.Looper.loop(Looper.java:344)
	at android.app.ActivityThread.main(ActivityThread.java:8212)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:584)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1034)
```

**Comment by lm41** [comment]
This was fixed with ad7ffb68aeb9fb9716a204426b4d0b0ba8c9ad63. Can you try the newest beta?

**Comment by Racooder** [comment]
I'm using the newest beta and it works. But restarting the app without beta solved the problem too.
